pytest.ini

```diff
@@ -1,5 +1,2 @@
 [pytest]
-addopts = -s -v --runslow --junitxml=./reports/pytest-results.xml --strict-markers --durations=0 --cov=nucypher --cov-report xml:reports/coverage.xml
-markers =
-    slow: marks tests as slow (skipped by default, use '--runslow' to include these tests)
-junit_family = xunit1
+addopts = --capture fd --maxfail 1
```

Comment:
What does these options mean?

Reply:
Feel free to alter these on your system to tweak the way pytest runs:

 `--capture fd` means *take stdout and 1 and 2 and redirect to a temp file (<https://docs.pytest.org/en/stable/capture.html#setting-capturing-methods-or-disabling-capturing>).  This is important if you want to see the console output of CLI tests while still being able to hit a debugging breakpoint.  The shorthand is `-s`.

`--maxfail 1` means - *Fail and exit pytest after 1 failure.*